plugins/mimecast/plugin.spec.yaml

```diff
@@ -400,78 +864,19 @@ actions:
     title: Get Managed URL
     description: Get information on a managed URL
     input:
-      domain:
-        title: Domain
-        description: The managed domain
-        type: string
-        example: rapid7.com
-        required: false
-      disable_user_awareness:
-        title: 'Filter: User Awareness'
-        description: Filter on whether or not User Awareness challenges for this URL
-        type: string
-        example: false
-        required: false
-        default: none
-        enum:
-        - none
-        - 'false'
-        - 'true'
-      disable_log_click:
-        title: 'Filter: Log Click'
-        description: Filter on whether or not clicks are logged for this URL
-        type: string
-        required: false
-        example: true
-        default: none
-        enum:
-        - none
```

Comment:
Why do you remove most of the filter inputs?

Reply:
I checked the API documentation and noticed that these parameters are not used. I had a second look and the action itself further filters the results by this fields. At the time of writing I had not paid attention to this. You are right I will roll over these changes and add the  unit tests